scripts/create-package.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/sh
+
+set -x
```

Comment:
Suggest making the `-x` activate after positive usage is confirmed?

Otherwise, on first invocation:


![2021-11-25_09-56](https://user-images.githubusercontent.com/1557255/143485491-08c656d3-ae46-4260-ac3c-5f15663f93ca.png)